### ISSUE TITLE
[v2020.08.07] Monsters don't turn around anymore (WhatIsThat Script)

### ISSUE THREAD

**Issue by DV666** [Albeoris/Memoria#150]
**Memoria version :** v2020.08.07 (last)

Hello team,

I think i found another issue (minor) about **GetDirection(unit) from btl_mot** : when you flip monsters with What Is That : as soon as you attack them or if they attack, they don't come back to face the players.
They didn't rotate and keeps looking in the same direction, which is the same as the characters.

Code us by **What Is That** :

```
namespace Memoria.Scripts.Battle
{
    /// <summary>
    /// What’s That!?
    /// </summary>
    [BattleScript(Id)]
    public sealed class WhatIsThatScript : IBattleScript
    {
        public const Int32 Id = 0054;

        private readonly BattleCalculator _v;

        public WhatIsThatScript(BattleCalculator v)
        {
            _v = v;
        }

        public void Perform()
        {
            if (!_v.Target.IsPlayer && BattleState.IsSpecialStart)
            {
                _v.Context.Flags |= BattleCalcFlags.Miss;
                return;
            }

            _v.Target.FaceAsUnit(_v.Caster); <== This part.
            _v.Target.ChangeRowToDefault();
        }
    }
}
```

**FaceAsUnit :**  

	
```
int direction = btl_mot.GetDirection(unit); <=== The issue is here.
this.Data.evt.rotBattle.eulerAngles = new Vector3(this.Data.evt.rotBattle.eulerAngles.x, (float)direction, this.Data.evt.rotBattle.eulerAngles.z);
this.Data.rot.eulerAngles = new Vector3(this.Data.rot.eulerAngles.x, (float)direction, this.Data.rot.eulerAngles.z);

```

**GetDirection :** 

```
		public static int GetDirection(BattleUnit btl)
		{
			return (int)btl.Data.evt.rotBattle.eulerAngles[1];
		}
```

Seem (int)btl.Data.evt.rotBattle.eulerAngles[1]; doesn't work : I'm not comfortable with C# so i restore with previous code from previous Memoria version : 

```
		public static int GetDirection(BattleUnit btl)
		{
			if (!btl.IsPlayer)
			{
				return 0;
			}
			return 180;
		}
```

And now, that's works.

**Comment by Albeoris** [comment]
Feel free to send pull-requests! :)

**Comment by Tirlititi** [comment]
Fixed with commit 3810875
